pages/api/tool/getOne.ts

```diff
@@ -0,0 +1,15 @@
+import { Tool } from "database/models/tool";
+import { NextApiRequest, NextApiResponse } from "next";
+
+const getOne = async (
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> => {
+    const { id } = req.query;
+
+    await Tool.findById(id)
+        .then((tool) => res.status(200).json(tool))
+        .catch((err) => res.status(500).send(err));
```

Comment:
will this throw an error if no tool for the given id is found?

Reply:
it won't - I'll change it to return 404 if no tools have the given id